tests/unit/modules/default/calendar/calendar_fetcher_utils_spec.js

```diff
@@ -49,5 +52,90 @@ describe("Calendar fetcher utils test", () => {
 			expect(filteredEvents[0].title).toBe("ongoingEvent");
 			expect(filteredEvents[1].title).toBe("upcomingEvent");
 		});
+
+		it("should return the correct times when recurring events pass through daylight saving time", () => {
+			const data = ical.parseICS(`BEGIN:VEVENT
+DTSTART;TZID=Europe/Amsterdam:20250311T090000
+DTEND;TZID=Europe/Amsterdam:20250311T091500
+RRULE:FREQ=WEEKLY;BYDAY=FR,MO,TH,TU,WE,SA,SU
+DTSTAMP:20250531T091103Z
+ORGANIZER;CN=test:mailto:test@test.com
+UID:67e65a1d-b889-4451-8cab-5518cecb9c66
+CREATED:20230111T114612Z
+DESCRIPTION:Test
+LAST-MODIFIED:20250528T071312Z
+SEQUENCE:1
+STATUS:CONFIRMED
+SUMMARY:Test
+TRANSP:OPAQUE
+END:VEVENT`);
+
+			const filteredEvents = CalendarFetcherUtils.filterEvents(data, defaultConfig);
+
+			console.log(filteredEvents);
+
+			const januaryFirst = filteredEvents.filter((event) => moment(event.startDate, "x").format("MM-DD") === "01-01");
+			const julyFirst = filteredEvents.filter((event) => moment(event.startDate, "x").format("MM-DD") === "07-01");
+
+			let januaryMoment = moment(`${moment(januaryFirst[0].startDate, "x").format("YYYY")}-01-01T09:00:00`)
+				.tz("Europe/Amsterdam", true) // Convert to Europe/Amsterdam timezone (see event ical) but keep 9 o'clock
+				.tz(moment.tz.guess()); // Convert to guessed timezone as that is used in the filterEvents
+
+			let julyMoment = moment(`${moment(julyFirst[0].startDate, "x").format("YYYY")}-07-01T09:00:00`)
+				.tz("Europe/Amsterdam", true) // Convert to Europe/Amsterdam timezone (see event ical) but keep 9 o'clock
+				.tz(moment.tz.guess()); // Convert to guessed timezone as that is used in the filterEvents
+
+			expect(januaryFirst[0].startDate).toEqual(januaryMoment.format("x"));
+			expect(julyFirst[0].startDate).toEqual(julyMoment.format("x"));
+		});
+
+		it("should return the correct moments based on the timezone given", () => {
+			const data = ical.parseICS(`BEGIN:VEVENT
+DTSTART;TZID=Europe/Amsterdam:20250311T090000
+DTEND;TZID=Europe/Amsterdam:20250311T091500
+RRULE:FREQ=WEEKLY;BYDAY=FR,MO,TH,TU,WE,SA,SU
+DTSTAMP:20250531T091103Z
+ORGANIZER;CN=test:mailto:test@test.com
+UID:67e65a1d-b889-4451-8cab-5518cecb9c66
+CREATED:20230111T114612Z
+DESCRIPTION:Test
+LAST-MODIFIED:20250528T071312Z
+SEQUENCE:1
+STATUS:CONFIRMED
+SUMMARY:Test
+TRANSP:OPAQUE
+END:VEVENT`);
+
+			const moments = CalendarFetcherUtils.getMomentsFromRecurringEvent(data["67e65a1d-b889-4451-8cab-5518cecb9c66"], moment(), moment().add(365, "days"));
+
+			const januaryFirst = moments.filter((m) => m.format("MM-DD") === "01-01");
+			const julyFirst = moments.filter((m) => m.format("MM-DD") === "07-01");
+
+			expect(januaryFirst[0].toISOString(true)).toContain("09:00:00.000+01:00");
+			expect(julyFirst[0].toISOString(true)).toContain("09:00:00.000+02:00");
+		});
+
+		it("debug", () => {
```

Comment:
ESLint complains that this test has no assertion. Shall we allow this and disable the ESLint rule for this test?

Reply:
I think it should be changed to make the assertion

Reply:
Ah sorry, this was left there by accident